Formula/python-build.rb

```diff
@@ -0,0 +1,44 @@
+class PythonBuild < Formula
```

Comment:
Not sure of best name. Basic Homebrew style would be `build`, but seemed a bit confusing and vague.

`python-build` was option I used here since it matched `python-tabulate` and most other repositories in https://repology.org/project/python:build/versions though there is an (unmaintained?) PyPI package with same name.

Other options would be name of org (e.g. `pypa-build`) or name of extra executable installed (e.g. `pyproject-build`) but wasn't too sure about either.

Reply:
Maybe we can have both, with an alias, if necessary. I am fine `python-build`.

Reply:
I personally prefer `pypa-build` (in case there would be other `py*-build` in the future) if matters